powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java

```diff
@@ -96,9 +98,11 @@ public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createP
      */
     @PostMapping("/offline/non-personalized/create")
     public ObjectResponse<CreateNonPersonalizedOfflineSignaturePayloadResponse> createNonPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreateNonPersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
-        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest received: {}", request);
+        logger.info("CreateNonPersonalizedOfflineSignaturePayloadRequest received.");
```

Comment:
the same refactor as above
use action: CreateNonPersonalizedOfflineSignaturePayloadRequest, state: received and identifier relevant to the request